barretenberg/cpp/pil/vm2/execution.pil

```diff
@@ -26,16 +26,38 @@ include "nullifier_check.pil";
 namespace execution;
 
 pol commit sel; // subtrace selector
+// Subtrace operation id
+pol commit subtrace_operation_id;
+
+// Subtrace Dispatch selectors
+pol commit alu_sel;
+pol commit bitwise_sel;
+pol commit to_radix_sel;
+pol commit ecc_add_sel;
+pol commit poseidon2_perm_sel;
 
 pol commit ex_opcode;
 pol commit indirect;
 // operands
 pol commit op1, op2, op3, op4;
+pol commit op5, op6, op7;
 // resolved operands
 pol commit rop1, rop2, rop3, rop4;
+pol commit rop5, rop6, rop7;
+// Registers
+pol commit reg1, reg2, reg3, reg4;
+pol commit reg5, reg6, reg7;
```

Comment:
These many registers are probably overkill and unnecessary - but we'll keep them here for now until we're sure we don't need them